src/arrayMethodSort.js

```diff
@@ -4,8 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
instead of that if statement let's just specify the default value
```suggestion
  [].__proto__.sort2 = function (compareFunction = ........) {
```